### ISSUE TITLE
Estrange parseErr 

### ISSUE THREAD

**Issue by ivanquirino** [niftylettuce/frisbee#50]
When I test this:
```javascript
const client = new Frisbee({baseURI: 'http://localhost:3000'});
console.log(client);   
```
I've got this:
```
  console.log src\sync\tests\sync-http-client.test.js:13
    Frisbee {
      opts: { baseURI: 'http://localhost:3000' },
      parseErr:
       Error: Invalid JSON received from http://localhost:3000
```

I haven't done any requests before calling that code, so I must assume this is some kind of bug?

**Comment by OmgImAlexis** [comment]
```console
> const Frisbee = require('frisbee')
undefined
> const client = new Frisbee({baseURI: 'http://localhost:3000'});
undefined
> client
Frisbee {
  opts: { baseURI: 'http://localhost:3000' },
  parseErr: Error: Invalid JSON received from http://localhost:3000
    at new Frisbee (/Users/xo/code/bree/node_modules/frisbee/lib/frisbee.js:137:21)
    at repl:1:16
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:440:10)
    at emitOne (events.js:120:20)
    at REPLServer.emit (events.js:210:7)
    at REPLServer.Interface._onLine (readline.js:282:10),
  headers: {},
  arrayFormat: 'indices',
  get: [Function],
  head: [Function],
  post: [Function],
  put: [Function],
  del: [Function],
  options: [Function],
  patch: [Function],
  interceptor: 
   Interceptor {
     interceptedMethod: [Function],
     register: [Function],
     unregister: [Function],
     clear: [Function],
     interceptors: [] } }
```

**Comment by OmgImAlexis** [comment]
That actually isn't an error it's the error that's bound to the Frisbee class.

@niftylettuce is there anyway we can hide that error from the class?

**Comment by pablopunk** [comment]
It could be a `const` inside `frisbee.js`

```js
const parseErr = new Error('...')

export default class Frisbee {
}
```

I don't think it's worth to export the error in the module

**Comment by niftylettuce** [comment]
Fixed in version 2.0.3, please install via `npm install frisbee@latest` or `yarn add frisbee@latest`
